OWNERS_ALIASES

```diff
@@ -147,14 +147,14 @@ aliases:
 
   sig-scheduling-maintainers:
     - alculquicondor
-    - bsalamat
     - k82cn
     - wojtek-t
     - ravisantoshgudimetla
     - Huang-Wei
     - ahg-g
+  # emeritus:
+  # - bsalamat
```

Comment:
Note, this mimics the style used with the `sig-cli-maintainers` alias:  https://github.com/kubernetes/kubernetes/blob/c1707541d8430716477f2dfa163ef9f9cdcbe9ba/OWNERS_ALIASES#L170-L184